### PR TITLE
176894722 setup light client

### PR DESCRIPTION
**Pull request by dehekkendekrekker**
This PR enables subtensor to be set up as a light node. This allows the chain the sync faster, as well as takes less space on disk.
- The docker node has been updated to run a light node on the akira network.
- 4 installation scripts have been made to setup subtensor as a systemd service.
Kusanagi/Akira Full/Light

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,19 @@
 
 * It is assumed your default outgoing traffic policy is ACCEPT. If not, make sure outbound traffic to port 30333 is allowed.
 
+## Node types
+Subtensor can be run as two types of nodes:
+1) as a FULL node
+2) as a LIGHT node
+
+### Running as a full node
+Running as a full node means that information about blocks older than 256 blocks is discarded, however
```
Comment (unconst) [inline README.md:33]: Shouldn't this be "running a light node" and "are discarded"

File: README.md [hunk 2]
```diff
@@ -24,6 +24,19 @@
...
+all extrinsics are kept. This means that a relatively large amount of data is stored on storage.
+For most users, this option is less preferred over running a light node.
+
+### Running as a light node
+A light node stores only the runtime and the current state, and therefore takes up less space
```
Comment (unconst) [inline README.md:38]: Ok, nvm, this makes sense now.

**Review by unconst — APPROVED** [review]
Beautiful PR, awesome stuff.

File: README.md (hunk 1 above)
Comment (dehekkendekrekker) [reply README.md:33]: No actually, what we thought of a full node is actually an archive node in substrate terminology. A full node is actually a bit lighter than an archive node.

**Pull request merged**